instant-xml/tests/all.rs

```diff
@@ -1,28 +1,17 @@
 use instant_xml::{Error, FromXml, ToXml};
-
-#[derive(Debug, Eq, PartialEq, ToXml)]
-struct Nested {
-    #[xml(namespace(bar))]
-    flag: bool,
-}
-
-#[derive(Debug, Eq, PartialEq, ToXml)]
-struct NestedWrongPrefix {
-    #[xml(namespace(dar))]
-    flag: bool,
-}
+//TODO: Add compile time errors check?
 
 #[derive(Debug, Eq, PartialEq, ToXml)]
 struct Unit;
 
 #[test]
 fn unit() {
-    assert_eq!(Unit.to_xml().unwrap(), "<Unit></Unit>");
+    assert_eq!(Unit.to_xml().unwrap(), "<Unit xmlns=\"\"></Unit>");
```

Comment:
We should only a serialize an element's default namespace if it's different from a parent's element default namespace (which is not the case here because there is no parent element).

Reply:
done